language-design.md

```diff
@@ -38,16 +76,43 @@ you know a poorly designed language? What are the symptoms?
 
 **Response**
 
+The first positive quality of a language that came to mind is consistency. Constructs that seem sematincally equivalent should behave the same way in the language. For example, consider the Java type system. `int` and `Object` are both types, but the Java generic system only allows for generic objects, despite seeming to by a construct concerning types. C# on the other hand has the same idea of primitive-ish type and reference types, but its generic system works for both. This is closely related to the principle of least surprise - it is nice if your PL behaves as users easonably expect it to.
+
+New versions of the language should also be backwards compatible with themselves. Introducing breaking changes into a language is an indication that the initial version had a design flaw that was serious and fundamental enough that it had to be fixed and could not be fixed without breaking code. An example of this was switch form Python 2 to 3.
```

Comment:
Python does provide a conversion script, [`2to3`](https://docs.python.org/2/library/2to3.html), which converts from Python 2 files to Python 3 files. This helps with the backwards incompatibility to a large amount.